cmd/etcd-launcher/main.go

```diff
@@ -290,8 +290,8 @@ func etcdCmd(config *config) []string {
 		fmt.Sprintf("--advertise-client-urls=https://%s.etcd.%s.svc.cluster.local:2379,https://%s:2379", config.podName, config.namespace, config.podIP),
 		fmt.Sprintf("--listen-client-urls=https://%s:2379,https://127.0.0.1:2379", config.podIP),
 		fmt.Sprintf("--listen-metrics-urls=http://%s:2378,http://127.0.0.1:2378", config.podIP),
-		fmt.Sprintf("--listen-peer-urls=https://%s:2380", config.podIP),
-		fmt.Sprintf("--initial-advertise-peer-urls=https://%s.etcd.%s.svc.cluster.local:2380", config.podName, config.namespace),
+		fmt.Sprintf("--listen-peer-urls=http://%s:2380", config.podIP),
```

Comment:
you removed support for secure peer communication at the moment, right?

Reply:
Yes, I just update the removed the other options :+1: